components/status-message/assets/StatusMessage.scss

```diff
@@ -0,0 +1,71 @@
+@import "@hods/sass-base";
+@import "govuk-frontend/govuk/components/button";
+
+$govuk-button-colour: govuk-colour("white");
+$govuk-button-text-colour: govuk-colour("black");
+
+.hods-status-message {
+  color: govuk-colour("white");
+  background-color: govuk-colour("blue");
+  @include govuk-responsive-padding(4, "all");
+
+  & p {
```

Comment:
Could this just be `p {` or am I being stupid?

Reply:
It did start off as just `p`, but I need the specificity for the style to be applied:
![image](https://user-images.githubusercontent.com/7834222/118467734-f8136800-b6fb-11eb-9681-c09fa95f9573.png)

Reply:
...actually it seems fine without... I'll ditch the parent part.